CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.17.0-rc.0 / 2020-03-11
+
+This release implements isolation in TSDB. API queries and recording rules are
+guaranteed to only see full scrapes and full recording rules. This comes with a
+certain overhead in resource usage. Depending on the situation, there might be a
```

Comment:
It sounds like "Inevitable you will see certain overhead". It really IMO sounds more negative then it is... I think we can tell this softer.